.storybook/main.ts

```diff
@@ -13,5 +13,22 @@ const config: StorybookConfig = {
   docs: {
     autodocs: "tag",
   },
+  staticDirs: [
+    // See https://github.com/storybookjs/storybook/tree/4f0c895bc53116272ef598f19e8d869213be49a9/code/frameworks/nextjs#nextfontlocal
+    {
+      from: "../src/app/fonts",
+      to: "src/app/fonts",
+    },
+  ],
+  async webpackFinal(config) {
+    config.module ??= {};
+    config.module.rules ??= [];
+    config.module.rules.push({
+      test: /\.ftl/,
+      type: "asset/source",
+    });
+
+    return config;
+  },
```

Comment:
This replicates https://github.com/mozilla/blurts-server/blob/fae90e0dbbfe38c0b92c413b0c46b34d7de3a455/next.config.js#L153-L156

It allows us to `require` `.ftl` files.